docs/user-guide/docs-source/index.md

```diff
@@ -1,37 +1,13 @@
 # Welcome to AWS Ambit Scenario Designer for Unreal Engine 4
 
-AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline content creation at scale for autonomous vehicle and robotics simulation applications.
+AWS Ambit Scenario Designer for Unreal Engine 4 (Ambit) is a suite of tools to streamline 3D content creation at scale for autonomous vehicle and robotics simulation applications. From procedural placement of obstacles to automatic generation of whole cities, Ambit can help you quickly create the 3D environments and scenarios you need for your simulations or other 3D applications.
 
-Currently Ambit can be installed as a plugin for [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
+![Ambit sample image](images/AmbitBanner.jpg)
 
-## Getting Started
+Ambit is implemented as an Unreal Engine plugin and has been tested for compatibility with [Unreal Engine 4.27](https://docs.unrealengine.com/en-US/Support/Builds/ReleaseNotes/4_27/index.html).
```

Comment:
We reference "UE4" throughout the document. We should preface this with the abbreviated form after Unreal Engine 4.27.
"Unreal Engine 4.27" => "Unreal Engine 4.27 (UE4)"